float64 rotateAngle
---
int64 status
